tests/Test/Proto/Internal.hs

```diff
@@ -0,0 +1,96 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Proto.Internal (testTree) where 
+
+import Data.Aeson qualified as Aeson
+import Data.Aeson.Encoding qualified as Aeson.Encoding
+-- import Data.Default (Default (..))
+import Data.List.NonEmpty (NonEmpty(..))
+
+import GHC.Exts (IsList (..))
+
+import Proto3.Suite.Class (HasDefault (..))
+import Proto3.Suite.JSONPB.Class 
+  ( defaultOptions
+  , object
+  , objectOrNull
+  , pairs
+  , pairsOrNull
+  )
+import Proto3.Suite.DotProto.AST (Path(..))
+import Proto3.Suite.DotProto.Generate (renameProtoFile)
+import Proto3.Suite.DotProto.Internal 
+  ( CompileError (..)
+  , toModulePath
+  )
+
+import Hedgehog (property, withTests, (===))
+
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.Hedgehog (testProperty)
+
+--------------------------------------------------------------------------------
+
+testTree :: TestTree
+testTree =
+  testGroup
+    "Internal"
```

Comment:
What does "Internal" mean here?  Does it mean you are testing details that are considered internal to the package, or does it mean something else?

Reply:
Its an arbitrary name. I always pick the name of the enclosing module.

Reply:
So it's referring to `Proto3.Suite.DotProto.Internal`?

Reply:
Yes

Reply:
OK, thanks.